pom.xml

```diff
@@ -3,17 +3,16 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>org.jvnet.hudson.plugins</groupId>
-        <artifactId>instant-messaging-parent</artifactId>
-        <version>1.29</version>
-        <relativePath>../instant-messaging-parent-plugin/pom.xml</relativePath>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>4.13</version>
     </parent>
-
+    <groupId>org.jvnet.hudson.plugins</groupId>
     <artifactId>instant-messaging</artifactId>
     <name>Jenkins instant-messaging plugin</name>
-    <version>1.40-SNAPSHOT</version>
```

Comment:
Strange, I'm sure this one was in variables. Could `mvn release:prepare release:publish` rewrite this into static strings?

Reply:
the plugin wasn't properly incrementalised (https://github.com/jenkinsci/incrementals-tools) before so release would've put it there, but it'll work now,